services/ui-src/src/components/export/ExportedReportBanner.tsx

```diff
@@ -0,0 +1,54 @@
+import { Box, Text, Button, Image } from "@chakra-ui/react";
+import pdfIcon from "assets/icons/pdf/icon_pdf_white.svg";
+import qmVerbiage from "verbiage/export/qm-export";
+
+export const ExportedReportBanner = () => {
+  const { reportBanner } = qmVerbiage;
+
+  const onClickHandler = () => {
+    window?.print();
+  };
+
+  return (
+    <Box data-testid="exportedReportBanner" sx={sx.container}>
```

Comment:
We can remove `data-testid` since we're moving away from using those for unit tests purposes.

Reply:
good to know, removed!